### ISSUE TITLE
Steam TOTP codes broken since 4.0.7

### ISSUE THREAD

**Issue by klausenbusk** [Yubico/yubikey-manager#455]
- YubiKey Manager (ykman) version: `YubiKey Manager (ykman) version: 4.0.7`
- How was it installed?: From Arch Linux's repository: https://archlinux.org/packages/community/any/yubikey-manager/
- Operating system and version: Arch Linux
- YubiKey model and version: YubiKey 4
- Bug description summary: `ykman` doesn't return special Steam alphanum TOTP codes anymore

Steps to reproduce
---

`$ ykman oath accounts code Steam:foobar`

Expected result
---

```sh
$ ykman oath accounts code Steam:foobar
Touch your YubiKey...
Steam:foobar <alphanum TOTP code>
```

Actual results and logs
---

```sh
$ ykman oath accounts code Steam:foobar
Touch your YubiKey...
Steam:foobar <normal TOTP code>
```

Other info
---

Reverting https://github.com/Yubico/yubikey-manager/commit/a94a1d40eb400496b0e04e670d168a10dd87f795 fixes the issue.

/cc @dainnilsson

**Comment by dainnilsson** [comment]
That's annoying, I swear we tested this... Anyway, a fix has been committed in 849f4f004 which will be included in the next release. As a workaround for now, you can use `ykman oath accounts code --single Steam:foobar`, which will give you the correct code.

**Comment by jiawen** [comment]
Is this fixed / worked around in the 4.0.8 release? If so we can close this.

**Comment by MaienM** [comment]
This appears to work correctly in 4.0.8, yes.
